chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -49,17 +49,23 @@ touch $RUNFILE
 trap cleanup INT EXIT
 
 <% if @use_ha %>
-# Shutdown pacemaker so the remaining OpenStack services are stopped
+# Shutdown corosync so the remaining OpenStack services are stopped
 # This is needed so that the zypper dup won't trigger any db migrations on package update
-service pacemaker stop
+service corosync stop
 
 ret=$?
 if [ $ret != 0 ] ; then
-    log "Error occured during shutting down pacemaker service"
+    log "Error occured during shutting down corosync service"
     echo $ret > $UPGRADEDIR/crowbar-pre-upgrade-failed
     exit $ret
 fi
 
+# Make sure sbd is shut down (if it was configured) before proceeding with next actions
+while systemctl is-active sbd >/dev/null ; do
```

Comment:
I'd prefer ```systemctl --quiet is-active corosync``` instead of the redirection.

I'd also say we need to error out here if sbd is stopped after a certain timeout (1-2minutes?)